fr.json

```diff
@@ -1,7 +1,7 @@
 {
   "FR": "Libre",
-  "FR4": "3 TN",
-  "GL": "Objectif",
+  "FR4": "3 points de virage max",
```

Comment:
Can we use 3 pvm or 3 points or something similar? There is no space in the UI for more than two short words.